.github/workflows/linter.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
       - name: Install build dependencies
         run: |
           sudo apt-get install -y /tmp/cvmfs-build-deps_2.1.0_all.deb
+          sudo apt-get install -y libacl1-dev #TODO: figure out why not installed with mk-build-deps
```

Comment:
@vvolkl  `COMPARE_TO_LIBACL` checking functionality still requires *patched* libacl. So not sure why you are adding this dependency, I am not aware of any usage of libacl beyond `marshal_xattrs()`.

Reply:
Right, that's  true. I'll just remove it from the build system then. I'll set up a separate test environment for those tests.